app/assets/javascripts/discourse/app/routes/preferences-interface.js

```diff
@@ -9,9 +10,14 @@ export default RestrictedUserRoute.extend({
       model: user,
       textSize: user.get("currentTextSize"),
       themeId: currentThemeId(),
+      userColorSchemeId:
+        parseInt(Session.currentProp("userColorSchemeId"), 10) || null,
```

Comment:
You should be able to say `this.session.userColorSchemeId` without importing it.